.github/workflows/self-scheduled.yml

```diff
@@ -306,7 +323,7 @@ jobs:
           TF_CPP_MIN_LOG_LEVEL=3 python -c "import tensorflow as tf; print('TF GPUs available:', bool(tf.config.list_physical_devices('GPU')))"
           TF_CPP_MIN_LOG_LEVEL=3 python -c "import tensorflow as tf; print('Number of TF GPUs available:', len(tf.config.list_physical_devices('GPU')))"
 
-      - name: Run all tests on GPU
+      - name: Run all tests on multi-GPU
```

Comment:
```suggestion
      - name: Run tests/ on multi-GPU
```

Reply:
But it is RUN_SLOW=1 - so everywhere else in config it says "all tests" for this env var.